test/edges/edges_test.go

```diff
@@ -45,3 +47,90 @@ func TestEdges(t *testing.T) {
 		t.Errorf("Expected output:\n%s\nactual:\n%s", b.String(), out)
 	}
 }
+
+// TestDirectEdges deploys a terminus and then generates a load generator which
+// sends traffic directly to the pod ip of the terminus pod. Traffic which is
+// addressed this way (as opposed to using the service name) does not show up
+// in the `linkerd edges` command. This test should be updated once
+// `linkerd edges` is updated to support this kind of traffic.
```

Comment:
For my own understanding, this will likely be a short-lived comment as you are actively working on this, correct?

Reply:
Laying the groundwork for it, yeah.  The feature is targeted for 2.7